lib/docker-sync/sync_strategy/unison-unox.rb

```diff
@@ -105,7 +106,7 @@ def start_container
           exists = `docker ps --filter "status=exited" --filter "name=#{container_name}" | grep #{container_name}`
           if exists == ''
             say_status 'ok', "creating #{container_name} container", :white if @options['verbose']
-            cmd = "docker run -p '#{@options['sync_host_port']}:#{UNISON_CONTAINER_PORT}' \
+            cmd = "docker run -p '127.0.0.1::#{UNISON_CONTAINER_PORT}' \
```

Comment:
This is intended, I don't specify the host port that should be on the middle of the pattern `ip:host_port:container_port`